### PR TITLE
Fixes #2 - Escape input before sending to GitHub

### LINKED ISSUE #2: Escape input
http://flask.pocoo.org/docs/1.0/api/#flask.escape

### PR DESCRIPTION
**Pull request by miketaylr**
r? @karlcow

### CODE REVIEW

**Comment by karlcow** [comment]
whoops tests. :)

File: app/endpoints/issues.py [hunk 1]
```diff
@@ -5,6 +5,7 @@
 
 import re
 
+from cgi import escape
```
Comment (karlcow) [inline app/endpoints/issues.py]: I wonder about  using [`flask.escape`](http://flask.pocoo.org/docs/1.0/api/?highlight=escape#flask.escape) over `cgi.escape`

```python
>>> from flask import escape
>>> escape(None)
Markup(u'None')
>>> escape('')
Markup(u'')
>>> escape('<why> not?')
Markup(u'&lt;why&gt; not?')
>>> escape('<why style="foo"> not?')
Markup(u'&lt;why style=&#34;foo&#34;&gt; not?')
>>> print(escape('<why> not?'))
&lt;why&gt; not?
>>> foo = escape('<why> not?')
>>> foo
Markup(u'&lt;why&gt; not?')
>>> dir(foo)
['__add__', '__class__', '__contains__', '__delattr__', '__doc__', '__eq__', '__format__', '__ge__', '__getattribute__', '__getitem__', '__getnewargs__', '__getslice__', '__gt__', '__hash__', '__html__', '__html_format__', '__init__', '__le__', '__len__', '__lt__', '__mod__', '__module__', '__mul__', '__ne__', '__new__', '__radd__', '__reduce__', '__reduce_ex__', '__repr__', '__rmod__', '__rmul__', '__setattr__', '__sizeof__', '__slots__', '__str__', '__subclasshook__', '_formatter_field_name_split', '_formatter_parser', 'capitalize', 'center', 'count', 'decode', 'encode', 'endswith', 'escape', 'expandtabs', 'find', 'format', 'index', 'isalnum', 'isalpha', 'isdecimal', 'isdigit', 'islower', 'isnumeric', 'isspace', 'istitle', 'isupper', 'join', 'ljust', 'lower', 'lstrip', 'partition', 'replace', 'rfind', 'rindex', 'rjust', 'rpartition', 'rsplit', 'rstrip', 'split', 'splitlines', 'startswith', 'strip', 'striptags', 'swapcase', 'title', 'translate', 'unescape', 'upper', 'zfill']
```

**Comment by karlcow** [comment]
Just a minor comment about using a different escape. Good to go as-is.

**Comment by karlcow** [comment]
I let you merge if you want to act on the comment.

File: app/endpoints/issues.py (hunk 1 above)
Comment (miketaylr) [reply app/endpoints/issues.py]: Thanks, yeah. I had originally used `flask.escape` and can't 100% remember why I ended up with `cgi.escape`.

Maybe let's go back to flask to keep dependencies flatter.

**Pull request merged**